include/archi/chips/astral-cluster/pulp.h

```diff
@@ -34,7 +34,7 @@
 #include "archi/itc/itc_v1.h"
 #include "archi/riscv/priv_1_10.h"
 
-#include "archi/chips/carfield-cluster/memory_map.h"
+#include "archi/chips/astral-cluster/memory_map.h"
 #include "archi/chips/carfield-cluster/apb_soc.h"
```

Comment:
I don't know if also this line needs to be changed

Reply:
Yes!

Reply:
Also line 38 (the apb_soc.h include)!